pkg/operator/resources/operator/operator.go

```diff
@@ -390,6 +390,10 @@ func createControllerEnv(pullPolicy string) []corev1.EnvVar {
 		{
 			Name: "OS_CONFIGMAP_NAME",
 		},
+		{
+			Name:  "IMPORT_POD_RESTART_TOLERANCE",
```

Comment:
Do we want to set this variable in HCO or are we fine with the default (3)?

Reply:
I think we are fine with 3 as default. I will remove.

Reply:
@pkliczewski Regarding HCO, what will be the case of the env-variables values after upgrading the operator ? will they be preserved or the user will have to re-define them ? points to consider: some of the env-variables could be irrelevant after the upgrade. what's the convention for other deployments ?

Reply:
It depends on how we will implement it in the operator. ATM we will use what is there in the new version and old is forgotten.